### DISCUSSION TITLE
Maximum steering angle setting

### DISCUSSION THREAD

**Discussion by Lilly-yang** [prl-mushr/mushr#91] [Q&A]
Hi,

Currently, I know that I can adjust the maximum steering angle by [steering_angle_to_servo_gain](https://github.com/prl-mushr/vesc/blob/c792756d907ebda162133e3ededb1b66046a4a6d/vesc_main/config/racecar-uw-nano/vesc.yaml#L16)  in vesc.yaml, but this is based on the default maximum steering angle of 0.32. If I want to change the maximum steering angle (say 0.64), what parameters should I change?

Synchronously, how do I change the max steering angle in mushr_sim.

Thank you!

**Comment by schmittlema** [comment]
That can be done by setting[ `servo_min` and `servo_max` ](https://github.com/prl-mushr/vesc/blob/c792756d907ebda162133e3ededb1b66046a4a6d/vesc_main/config/racecar-uw-nano/vesc.yaml#L38-L39)to be 0.5 +/- the max steering angle you would like.  As a safety note, there is a hardware limitation on max steering angle and why we set it to the current values. If you exceed 0.32 you may make the servo try to turn harder than the steering system will allow and break something. The sim should be fine and reads from the same file for max steering angle.
